packages/netlify-cms-core/src/actions/config.js

```diff
@@ -84,6 +115,44 @@ export function applyDefaults(config) {
     });
 }
 
+export function addLocaleFields(fields, locales) {
+  const defaultLocale = locales[0];
+  const stripedFields = stripNonTranslatableFields(fields);
+  return locales.reduce((acc, item) => {
+    const selectedFields = item === defaultLocale ? fields : stripedFields;
+    return acc.push(
+      fromJS({
+        label: item,
+        name: item,
+        widget: 'object',
+        fields: selectedFields,
+        multiContentId: Symbol.for('multiContentId'),
+      }),
+    );
+  }, List());
+}
+
+function stripNonTranslatableFields(fields) {
+  return fields.reduce((acc, item) => {
+    const subfields = item.get('field') || item.get('fields');
+    const widget = item.get('widget');
+
+    if (List.isList(subfields)) {
+      return acc.push(item.set('fields', stripNonTranslatableFields(subfields)));
+    }
+
+    if (Map.isMap(subfields)) {
+      return acc.push(item.set('field', stripNonTranslatableFields([subfields])));
+    }
+
+    if (!NON_TRANSLATABLE_FIELDS.includes(widget)) {
```

Comment:
I think the better option would be to add a `translatable` flag to a field with sensible defaults for each field widget type. 

I could think of cases where I would want to for example "translate" a map type field or not translate a relation or a string.